docs.md

```
@@ -1,142 +1,141 @@
 
 
-<hr /><a id="class-phpdocsmdfunctionfinder"></a>
+<hr /><a id="class-phpdocsmd-mdtablegenerator"></a>
 
-### Class: \PHPDocsMD\FunctionFinder
+### Class: \PHPDocsMD\MDTableGenerator
 
-> Find a specific function in a class or an array of classes
+> Class that can create a markdown-formatted table describing class functions referred to via FunctionEntity objects
+
+###### Example
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h6

(MD001, heading-increment)